keep-ui/app/ai/ai.tsx

```diff
@@ -0,0 +1,152 @@
+"use client";
+import { Card, List, ListItem, Title, Subtitle } from "@tremor/react";
+import { useAIStats } from "utils/hooks/useAIStats";
+import { useSession } from "next-auth/react";
+import { getApiURL } from "utils/apiUrl";
+import { toast } from "react-toastify";
+import { useEffect, useState, useRef, FormEvent } from "react";
+
+export default function Ai() {
+  const { data: aistats, isLoading } = useAIStats();
```

Comment:
i dunno what's going to be in the future but I feel like adding this new endpoint is "redundant" now, because it only uses "existing data"
You can combine useAlerts and useIncidents to present this information. Regarding is_mining_enabled - seems like it can be in the config (if it's not per tenant atm)

Reply:
not that critical but just a bit more code to maintain rather than using existing